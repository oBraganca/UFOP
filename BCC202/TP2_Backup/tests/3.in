23
1 3
0 6
3 0
0 2
4 3
0 4
6 5
5 3
4 5
5 5
2 2
4 1
1 5
3 3
3 2
0 0
6 4
3 6
4 4
2 4
0 1
1 2
1 1
0
