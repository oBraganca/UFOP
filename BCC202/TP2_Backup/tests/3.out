Test 1:
YES
13|30|02|22|23|36|60|04|45|55|51|12|24|43|33|35|56|64|44|41|11|10|00

